dev_tools/nyaml2nxdl/nyaml2nxdl.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python3
+"""Main file of yaml2nxdl tool.
```

Comment:
Update name

Reply:
Not clear? `nyaml2nxdl` is the converter or tool name!

Reply:
Why is `yaml2nxdl` mentioned?